### DISCUSSION TITLE
refresh size/duration of multiple/all archives in one click

### DISCUSSION THREAD

**Discussion by nbehrnd** [borgbase/vorta#1358] [Ideas]
I unlinked a borg backup repository and eventually added it again to vorta because of my assumption of a local error.  (As it turns out, the freeze already was documented [elsewhere](https://github.com/borgbase/vorta/issues/1356).  Hints provided resolved the issue in question.).

However, for the repository reconnected to vorta, tab _archives_ listed only the dates of recording and the names of the individual archives.  To access the individual archive's size and the duration required to generate the archive in question, I had to 1) select the corresponding line, and 2) click on the refresh button (A) on the right hand side.  I observed that the selection of one line, and a click of the refresh button (B) below the table did not update the line to the effect to then equally display the criteria size and duration.

![suggestion_vorta](https://user-images.githubusercontent.com/19709084/173785102-32ceba89-b785-4fba-8cf5-daa7fde56bf2.png)

Ideally, it would be possible to select multiple (maybe all) lines to refresh with button (A) the display of size and duration about  multiple/all archives in one step.

**Comment by real-yfprojects** [comment]
This is easily possible as this type of change was already done for the delete button.

**Reply by real-yfprojects** [reply]
This is tracked in #1367.
